databases/week1/homewrk.sql

```diff
@@ -0,0 +1,82 @@
+Use hyf_lesson1;
+/* Find out how many tasks are in the task table */
+select
+  COUNT(id)
+from
+  task;
+  /* Find out how many tasks in the task table do not have a valid due date */
+select
+  COUNT(id)
+from
+  task
+where
+  IsNull(due_date);
+  /* Find all the tasks that are marked as done */
+select
```

Comment:
Here, imagine you don't know that status 3 = done. 
How would you approach it then? 

Hint:  You have to work with 2 tables. ✌️